docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,61 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE LOCAL
+    SET TIME ZONE expression
+
+Description
+-----------
+
+Sets the current default time zone displacement for the current SQL-session.
+
+If the ```LOCAL`` option is specified, the time zone for the current SQL-session
+will be set to the original time zone of the current SQL-session.
```

Comment:
is set to the original time zone of the session. 

Need to add more info what that even means. E.g. the time zone set on the computer running e.g. the CLI or the one set in the JVM running a client that is using the JDBC driver or whatever.

Reply:
Also.. need to clarify if this is the default behavior

Reply:
```
If the ``LOCAL`` option is specified, the time zone for the current session
is set to the to the default time zone of the Java virtual machine on which
Trino server is running.
```

Reply:
the server? Not the client?

Reply:
Yes, you are right @mosabua . Sorry for the confusion. I just verified it via debugger and indeed the timezone is set based on the client's JVM (or based on `--timezone` CLI option).